pom.xml

```diff
@@ -21,7 +21,7 @@
     <groupId>com.hotels.beans</groupId>
     <artifactId>bean-utils-library-parent</artifactId>
     <url>https://github.com/HotelsDotCom/bull</url>
-    <version>2.0.0-SNAPSHOT</version>
+    <version>2.1.0-SNAPSHOT</version>
```

Comment:
there is no need to increase the major version as it has never been released

Reply:
done

Reply:
Ok, I thought the minor should be increased as this was a new feature, but you're right thanks for clarifying!